### ISSUE TITLE
Add support for parsing test specific config options

### ISSUE THREAD

**Issue by hellais** [ooni/probe-cli#39]
Currently we have several options in the config file which are not supported and are just ignored.

We should be adding support for:

* [ ] Specifying which tests should be disabled or enable
* [ ] Parsing the test list category code settings
* [ ] The collector and bouncer URL settings

We should also look into what settings are defined inside of: https://github.com/ooni/probe-cli/blob/master/config/settings.go#L124 and evaluate if in fact all are necessary and if not remove them.

This is a good opportunity to also document which ones we do not implement yet, see: 
https://github.com/ooni/probe-cli/issues/25.

Maybe related too: https://github.com/ooni/probe-cli/issues/24

**Comment by bassosimone** [comment]
@hellais can we double check this issue? I have a sense that you have simplified significantly the config and therefore this issue is much less relevant than before.

**Comment by hellais** [comment]
Yes it's true that we have significantly simplified the config file, but we still need to eventually support specifying the category code of which sites should be tested and which tests should be enabled and disabled when running.

I changed the title thought to reflect that.
